students/17th/raon/hyeyoon/user/views.py

```diff
@@ -0,0 +1,46 @@
+import json
+from django.views           import View
+from django.http            import JsonResponse, HttpResponse
+from django.core.validators import validate_email
+from django.core.exceptions import ValidationError
+
+from .models import Account
+
+min_length = 8
```

Comment:
```suggestion
MIN_LENGTH = 8
```
변수화 잘하셨는데! 이런 케이스는 상수로 선언하는게 좋습니다.

Reply:
아래 로직도 변경해주세요~!

Reply:
변경했습니다~ ㅎㅎ